frontend/src/app/utils/validators/task-validators.ts

```diff
@@ -0,0 +1,56 @@
+import { AbstractControl, ValidatorFn, Validators } from '@angular/forms';
+import { getEnd, getPeriodDays, getStart } from '@utils/form-helpers/task-form.helper';
+
+import { TaskRepeatingPeriod } from '#types/dashboard/tasks/enums/task-repeating-period';
+import { differenceInDays } from 'date-fns';
+
+export const getTitleValidators = () => [
+    Validators.required,
+    Validators.minLength(2),
+    Validators.maxLength(60)
+];
+
+export const getDescriptionValidators = () => [
```

Comment:
Try to always specify return type forfunctions so it is immediately clear what is it.

Reply:
Fixed